src/Discord/Builders/CommandAttributes.php

```diff
@@ -75,7 +75,7 @@ public function setName(string $name): self
             throw new \LengthException('Command name can be only up to 32 characters long.');
         }
 
-        if ($this->type == Command::CHAT_INPUT && preg_match('/^[-_\p{L}\p{N}\p{Devanagari}\p{Thai}]{1,32}$/u', $name) === 0) {
+        if (isset($this->type) && Command::CHAT_INPUT == Command::CHAT_INPUT && preg_match('/^[-_\p{L}\p{N}\p{Devanagari}\p{Thai}]{1,32}$/u', $name) === 0) {
```

Comment:
shouldn't that be ``&& $this->type == Command::CHAT_INPUT &&``

Reply:
Thanks, it was obvious bad auto merge